README.md

```diff
@@ -1174,6 +1184,17 @@ a $A $B=`echo $A`:
   echo $A $B
 ```
 
+When [export](#export) is set, all `just` variables are exported as environment variables.
+
+#### Loading Environment Variables from a `.env` File
+
+`just` will load environment variables from a `.env` file if [dotenv-load](#dotenv-load) is set. The variables in the file will be available as environment variables to the recipes. See [dotenv-integration](#dotenv-integration) for more information.
+
+#### Setting `just` Variables from Environments Variables
+
+Environment variables can be propagated to `just` variables using the functions `env_var()` and `env_var_or_default()`. These variables will be available as environment variables to the recipes.
+See [environment-variables](#environment-variables).
```

Comment:
There are two sections called "Environment Variables". Maybe we could rename
## Environment Variables
(after Setting Variables from the Command Line)
to something different, like "Getting and exporting environment variables") ?

Reply:
Ah, right, so linking doesn't work? How about "Getting and Setting Environment Variables" as the header for the section that starts with "Assignments prefixed with the export keyword…"?

Reply:
Linking worked, but it a bit harder to navigate in the table of content. I edited as you proposed